src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -113,6 +113,14 @@ void MulticopterLandDetector::_update_params()
 	param_get(_paramHandle.minManThrottle, &_params.minManThrottle);
 	param_get(_paramHandle.landSpeed, &_params.landSpeed);
 
+	if (_param_lndmc_z_vel_max.get() > _params.landSpeed) {
```

Comment:
`LNDMC_Z_VEL_MAX` is actually another parameter I suspect we could live without.

Typically it's the land speed threshold constraining the max climb rate. `max_climb_rate = math::min(land_speed_threshold * 0.5f, _param_lndmc_z_vel_max.get());`

The other case is for 2 seconds (LAND_DETECTOR_LAND_PHASE_TIME_US) after `landed` the max_climb_rate is set to 2.5 x LNDMC_Z_VEL_MAX. Wouldn't the land speed threshold work here as well?

Reply:
> Wouldn't the land speed threshold work here as well?

Probably, honestly that was there since before I first did any changes and I only preserved it.